src/main/webapp/assets/js/feedScript.js

```diff
@@ -921,3 +994,94 @@ function toggleLegend() {
     legend.style.display = 'none';
   }
 }
+
+/**
+ * Shows/Hides the filters to the user.
+ */
+function toggleFilters() {
+  const filterCard = document.getElementById('filter-card');
+
+  if (getComputedStyle(filterCard, null).display === 'none') {
+    filterCard.style.display = 'block';
+    toggleFiltersButton.style.backgroundColor = 'white';
+    toggleFiltersButton.style.color = 'var(--dark-maroon)';
+    toggleFiltersButton.style.border = '1px solid var(--dark-maroon)';
+  } else {
+    filterCard.style.display = 'none';
+    // Using '' makes the colors fall back to CSS defaults
+    toggleFiltersButton.style.backgroundColor = '';
+    toggleFiltersButton.style.color = '';
+    toggleFiltersButton.style.border = '';
+  }
+}
+
+/**
+ * Filters the posts the user sees on the page based
+ * on the current state of the filter inputs.
+ */
+function filterAndUpdatePagePosts() {
+  // Grab the filter inputs
+  const filters = {
+    numPeople: parseInt(document.getElementById('num-people').value),
```

Comment:
For parseInt, it is recommended to define the radix! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
Good catch! I had no idea that it doesn't default to base 10